src/Slick/Serve.hs

```diff
@@ -0,0 +1,121 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Slick.Serve ( runServerStatic
```

Comment:
Missing:
1. A single interface function to start both servers.
2. A single interface function to kill the servers when the process exits.

Since purpose of this module is to provide self-contained testing functionality, we do not need to expose WS and HTTP servers separately.

Reply:
So, here is a tricky thing, `runServerWS` not only runs websocket server but also an http server too. So we can consider it as (1). I wanted to have also separate options to run just http server for some used, but if you thing that superflusious I can remove `runServer` and `runServerStatic`

Reply:
Oh, then please rename them as well.
It looks confusing at the moment.

Reply:
Interface changed to 3 functions `serverStart`, `serverStop`, `serverHandler` last one needed for handling request coming to server. Due to nature of Wai there is no simple way to stop server, one need to track MVar and depend on it in order to kill server. Not sure if full example required in `example-site` as it creates additional layer.